### PR TITLE
Add Operator typing status

### PR DESCRIPTION
**Pull request by DmitryPR**

### CODE REVIEW

File: ios/SaleMove/InteractableClient.swift [hunk 1]
```diff
@@ -16,7 +16,15 @@ extension InteractableClient: Interactable {
     open var onEngagementRequest: RequestOfferBlock {
         return { [unowned self] answer in
             let context = VisitorContext(type: .page, url: "https://www.libertymutual.com")
```
Comment (wifilinker) [inline ios/SaleMove/InteractableClient.swift]: why do we have this url hardcoded? The swift code is not shared with the client, just the compiled version, right?

File: ios/SaleMove/InteractableClient.swift [hunk 2]
```diff
@@ -16,7 +16,15 @@ extension InteractableClient: Interactable {
...
-            answer(context, true)
+
+            let completion: SuccessBlock = { success, error in
+                if let reason = error?.reason {
+                    let payload: [AnyHashable: Any] = ["message": reason]
+                    InteractableEmmiter.emitEvent(withName: "error", andPayload: payload)
```
Comment (wifilinker) [inline ios/SaleMove/InteractableClient.swift]: this change does not seem to be related to the commit, please extract into a separate commit and describe why it is needed.

**Comment by wifilinker** [comment]
review finished

File: ios/SaleMove/InteractableClient.swift (hunk 1 above)
Comment (DmitryPR) [reply ios/SaleMove/InteractableClient.swift]: You are correct, this is a public repo and should not have any specific URL hardcoded. I added a more generic one.

File: ios/SaleMove/InteractableClient.swift [hunk 3]
```diff
@@ -16,13 +16,7 @@ extension InteractableClient: Interactable {
...
         return { [unowned self] answer in
             let context = VisitorContext(type: .page, url: "https://www.glia.com/")
-
-            let completion: SuccessBlock = { success, error in
-                if let reason = error?.reason {
-                    let payload: [AnyHashable: Any] = ["message": reason]
-                    InteractableEmmiter.emitEvent(withName: "error", andPayload: payload)
-                }
-            }
+            let completion: SuccessBlock = { _, _ in }
```
Comment (wifilinker) [inline ios/SaleMove/InteractableClient.swift]: is this correct? looks weird, but i'm not too familiar with swift

Comment (DmitryPR) [reply ios/SaleMove/InteractableClient.swift]: Yes this is an empty completion block for now
